### ISSUE TITLE
Development environment is not able to build packages

### ISSUE THREAD

**Issue by eduardoj** [openSUSE/open-build-service#14420]
# Issue Description

The development environment is not able to build packages.

Expected Result
============

It should be possible to build packages in the development environment.

How to Reproduce
===============
1. Set up the development environment.
2. Branch a package, for example `openSUSE.org:games / ctris`
3. Packages are not built.
4. Watch the logs from the `backend` container (with `docker-compose logs -f backend`) and see the errors:
   ```
   retrying watcher for https://api.opensuse.org/public
   retrying watcher for
   setting up watcher for
   setting up watcher for https://api.opensuse.org/public
   2023-06-01 14:47:36: [19]    waiting for an event...
   2023-06-01 14:47:36: [1856]  POST (172.18.0.3)      /lasteventsproxy?client=x86_64&remoteurl=&filter=project/&filter=repository//snapshot/x86_64&filter=package//aaa_base&filter=package//ctris
   2023-06-01 14:47:36: [1857]  POST (172.18.0.3)      /lasteventsproxy?client=x86_64&remoteurl=https://api.opensuse.org/public&filter=package/games/ctris
   2023-06-01 14:47:36: [14]    GET (AJAX)             /lasteventsproxy?client=x86_64&remoteurl=&filter=project/&filter=repository//snapshot/x86_64&filter=package//aaa_base&filter=package//ctris
   2023-06-01 14:47:36: [14]    GET (AJAX)             /lasteventsproxy?client=x86_64&remoteurl=https://api.opensuse.org/public&filter=package/games/ctris
   14: SSL_connect error: Resource temporarily unavailable
   response from watcher for https://api.opensuse.org/public
   500 remote error: 14  SSL_connect error  Resource temporarily unavailable (http://backend:5352/lasteventsproxy)
   retrying in 60 seconds
   response from watcher for
   400 remote error: bad uri  /lastevents (http://backend:5352/lasteventsproxy)
   retrying in 60 seconds
   2023-06-01 14:47:36: [19]    waiting for an event...
   ```

Further Information
===============
* Rolling back in the git commit history to before #14401 seems to fix the issue.
* Our development environment is documented in our wiki: https://github.com/openSUSE/open-build-service/wiki/Development-Environment-Overview

**Comment by hennevogel** [comment]
@mlschroe is there something we need to configure to not use ssl now?

**Comment by eduardoj** [comment]
I've rebased on master and now I'm able to build packages within the development environment.

This was most likely fixed by #14425.

Closing.
